### ISSUE TITLE
Add a vertical line to ind_plots to show dose times

### ISSUE THREAD

**Issue by sn248** [UUPharmacometrics/xpose#136]
Hi

I am wondering if there is an easy way to add vertical line to show dosing events. I am thinking we have to use `guide` which options to `geom_abline` but could not figure out the syntax.

Another question, in `dv_vs_pred` type of diagnostic plots, is there anyway to color the points based on a `factor`, i.e., I would like to highlight a few points based on a factor, but can't seem to do so.

Thanks a lot!

**Comment by guiastrennec** [comment]
@sn248 yes this is something that I have on my todo list, I was thinking vertical lines for dosing and shaded areas for 0 order infusions.

**Comment by sn248** [comment]
Thank you very much!
